gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/actions/ExcuseGradeAction.java

```diff
@@ -90,6 +90,8 @@ public ActionResponse handleEvent(final JsonNode params, final AjaxRequestTarget
 
         target.addChildren(page, FeedbackPanel.class);
 
+        final String siteId = params.get("siteId").asText();
+        final String gradebookUid = params.get("gUid").asText();
```

Comment:
Could the parameter be called `gradebookId`?

Reply:
It's gradebookUid, like on all other places